spec/openapi.yaml

```diff
@@ -1604,6 +1604,24 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/StorePaymentGateways"
+  "/v{version}/tax-policy/regions":
+    get:
+      tags:
+        - TaxPolicyRegionList
```

Comment:
```suggestion
        - TaxPolicyRegion
```
Should be renamed to `TaxPolicyRegion` instead so that both the `Single` and `Listing` request are grouped together:
![image](https://user-images.githubusercontent.com/2146611/122943309-334a2c00-d377-11eb-8510-fe0126e73cf7.png)